README.md

```diff
@@ -1,24 +1,27 @@
 # Ably Livestream Chat Demo
 
-A fullstack chat application demonstrating some of the features of the Ably Chat SDK. It uses:
+A fullstack chat application demonstrating some of the features of the Ably Chat SDK and associated Chat React hooks. It uses:
 
 * [`rooms`](https://ably.com/docs/chat/rooms) and [`messages`](https://ably.com/docs/chat/rooms/messages) for users to send and subscribe to chat messages, and [`history`](https://ably.com/docs/chat/rooms/history) to retrieve those that have been previously sent.
 * [`reactions`](https://ably.com/docs/chat/rooms/reactions) to enable users to send and receive room-level reactions.
 * [`occupancy`](https://ably.com/docs/chat/rooms/occupancy) to show the number of users connected to the room.
 * [`presence`](https://ably.com/docs/chat/rooms/presence) to synchronize video playback between users.
+* [`Chat React hooks`](https://github.com/ably/ably-chat-js/tree/main/src/react) to access and manage the various chat features above.
 * Postgres to demonstrate storing [`room`](https://ably.com/docs/chat/rooms) and user data.
 
 > Note: some elements of this application are purely for demonstration purposes, such as the video sync any chat bots.
 
 ## Built with
 
-- [Ably Chat SDK](https://github.com/ably/ably-chat-js)
+- [Ably Chat SDK and Chat React hooks](https://github.com/ably/ably-chat-js)
```

Comment:
Can we perhaps link this to the react folder as it's a sub part of the main chat sdk?